src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -49,11 +51,13 @@ public function iSeeTheOrder(OrderInterface $order): void
     }
 
     /**
+     * @Given I am browsing orders
      * @When I browse orders
      */
     public function iBrowseOrders(): void
     {
         $this->client->index(Resources::ORDERS);
+        $this->client->addFilter('order[number]', 'desc');
```

Comment:
Should be reverted.

Reply:
Yep, the default ordering should be changed if needed: https://api-platform.com/docs/core/default-order/#overriding-default-order